src/assets/styles/components/_filters.css

```diff
@@ -0,0 +1,149 @@
+
+[class*="--hide"] {
+    display: none;
+}
+
+.filters {
+    margin-block-end: 1.75rem;
+
+    inclusive-disclosure {
+        padding-inline: 0;
+    }
+
+    inclusive-disclosure * ul {
+        padding-inline: 1.25rem;;
+
+    }
+
+    inclusive-disclosure > button {
+        position: relative;
+        padding-inline-start: 3.5625rem;
+    }
+
+    inclusive-disclosure > button::after {
+        right: 1.25rem;
+    }
+    
+    inclusive-disclosure > button::before {
+        background-color: var(--fl-linkFgColor, var(--color-indigo-700));
+        block-size: calc(1.125rem * var(--fl-enhance-font-size-factor, 1.5));
+        content: "";
+        display: inline-block;
+        inline-size: calc(0.9375rem * var(--fl-enhance-font-size-factor, 1.5));
+        left: 1.25rem;
+        mask-position: center;
```

Comment:
This appears to be causing the icons to be clipped slightly, see:

<img width="50" alt="Topic icon with slight clipping on the top" src="https://github.com/user-attachments/assets/6f1f4158-4b7b-4765-99f8-977fa5192b21" />

I tried centring horizontally but not vertically and it looked better.

Reply:
It doesn't appear to be clipped on my side -- I tested in Chrome, Firefox and Safari as well.